src/components/DropDownMenu.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import Checked from '../images/iconfinder_done-01_186405.svg'
+export const DropDownMenu = ({ isOpen, setFormat, format }) => {
+  const formatOptions = [{
+    id: 1,
+    type: 'Plain Text'
+  }, {
+    id: 2,
+    type: 'HTML'
+  }]
```

Comment:
Let's pass the options in as props so that the DropDownMenu component remains generic. Otherwise it can only be used for the export dropdown.